src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Startup.cs

```diff
@@ -65,11 +65,15 @@ public void ConfigureServices(IServiceCollection services)
                 .AddIdentityServerJwt();
 #endif
 #if (OrganizationalAuth)
+#pragma warning disable CS0618 // Type or member is obsolete
             services.AddAuthentication(AzureADDefaults.BearerAuthenticationScheme)
                 .AddAzureADBearer(options => Configuration.Bind("AzureAd", options));
+#pragma warning restore CS0618 // Type or member is obsolete
 #elif (IndividualB2CAuth)
+#pragma warning disable CS0618 // Type or member is obsolete
             services.AddAuthentication(AzureADB2CDefaults.BearerAuthenticationScheme)
                 .AddAzureADB2CBearer(options => Configuration.Bind("AzureAdB2C", options));
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
I'm hoping this goes away based on our conversations with @jmprieur in favor of using Microsoft.Identity.Web here too.

Reply:
Yes, that's out of scope for this PR. @jmprieur is working on these changes and we'll update this template when it's available.

Reply:
yes. It's coming. @Jennyf19 has a first cut which is going to be improved today: https://github.com/AzureAD/microsoft-identity-web/tree/componentsWebAssembly2